Boundary Docs: Configure workers refactor (#3077)
hashicorp/web-unified-docs: content/boundary/v1.0.x/content/docs/session-recording/configuration/create-storage-bucket.mdx

   - **Access key ID**: (Required) The access key ID that AWS generates for the IAM user to use with the storage bucket.
   - **Secret access key**: (Required) The secret access key that AWS generates for the IAM user to use with this storage bucket.

>    - **Worker filter**: (Required) A filter expression that indicates which Boundary workers have access to the storage. The filter must match an existing worker to create a Boundary storage bucket. This should be a boolean expression. To have Boundary format the filter for you, toggle the **Show filter generator** switch. Refer to [Generate a filter in the Admin UI](/boundary/docs/workers/filters#generate-a-filter-in-the-admin-ui) for the procedure, or [the filter example for storage buckets](/boundary/docs/workers/filters#example-worker-filter-for-storage-buckets) to learn about worker tags and filters.

Reviewer: ```suggestion
   - **Worker filter**: (Required) A filter expression that indicates which Boundary workers have access to the storage. The filter must match an existing worker to create a Boundary storage bucket. It should be a boolean expression. To have Boundary format the filter for you, click the **Show filter generator** toggle. Refer to [Generate a filter in the Admin UI](/boundary/docs/workers/filters#generate-a-filter-in-the-admin-ui) for the procedure, or [the filter example for storage buckets](/boundary/docs/workers/filters#example-worker-filter-for-storage-buckets) to learn about worker tags and filters.
```

   For more information, refer to the AWS documentation for [Logging IAM and AWS STS API calls with AWS CloudTrail](https://docs.aws.amazon.com/IAM/latest/UserGuide/cloudtrail-integration.html).
   - **Role tags**: An object with key-value pair attributes that is passed when you assume an IAM role.
   For more information, refer to the AWS documentation for [Passing session tags in AWS STS](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_session-tags.html).

>    - **Worker filter**: (Required) A filter expression that indicates which Boundary workers have access to the storage. The filter must match an existing worker to create a Boundary storage bucket. This should be a boolean expression. To have Boundary format the filter for you, toggle the **Show filter generator** switch. Refer to [Generate a filter in the Admin UI](/boundary/docs/workers/filters#generate-a-filter-in-the-admin-ui) for the procedure, or [the filter example for storage buckets](/boundary/docs/workers/filters#example-worker-filter-for-storage-buckets) to learn about worker tags and filters.

Reviewer: ```suggestion
   - **Worker filter**: (Required) A filter expression that indicates which Boundary workers have access to the storage. The filter must match an existing worker to create a Boundary storage bucket. It should be a boolean expression. To have Boundary format the filter for you, click the **Show filter generator** toggle. Refer to [Generate a filter in the Admin UI](/boundary/docs/workers/filters#generate-a-filter-in-the-admin-ui) for the procedure, or [the filter example for storage buckets](/boundary/docs/workers/filters#example-worker-filter-for-storage-buckets) to learn about worker tags and filters.
```

   - **Region**: (Optional) The region to configure the storage bucket for.
   - **Access key ID** (Required): The MinIO service account's access key to use with this storage bucket.
   - **Secret access key** (Required): The MinIO service account's secret key to use with this storage bucket.

>    - **Worker filter**: (Required) A filter expression that indicates which Boundary workers have access to the storage. The filter must match an existing worker to create a Boundary storage bucket. This should be a boolean expression. To have Boundary format the filter for you, toggle the **Show filter generator** switch. Refer to [Generate a filter in the Admin UI](/boundary/docs/workers/filters#generate-a-filter-in-the-admin-ui) for the procedure, or [the filter example for storage buckets](/boundary/docs/workers/filters#example-worker-filter-for-storage-buckets) to learn about worker tags and filters.

Reviewer: ```suggestion
   - **Worker filter**: (Required) A filter expression that indicates which Boundary workers have access to the storage. The filter must match an existing worker to create a Boundary storage bucket. It should be a boolean expression. To have Boundary format the filter for you, click the **Show filter generator** toggle. Refer to [Generate a filter in the Admin UI](/boundary/docs/workers/filters#generate-a-filter-in-the-admin-ui) for the procedure, or [the filter example for storage buckets](/boundary/docs/workers/filters#example-worker-filter-for-storage-buckets) to learn about worker tags and filters.
```

   - **Region**: (Optional) The region to configure the storage bucket for.
   - **Access key ID** (Required): The storage provider's service account's access key to use with this storage bucket.
   - **Secret access key** (Required): The storage provider's service account's secret key to use with this storage bucket.

>    - **Worker filter**: (Required) A filter expression that indicates which Boundary workers have access to the storage. The filter must match an existing worker to create a Boundary storage bucket. This should be a boolean expression. To have Boundary format the filter for you, toggle the **Show filter generator** switch. Refer to [Generate a filter in the Admin UI](/boundary/docs/workers/filters#generate-a-filter-in-the-admin-ui) for the procedure, or [the filter example for storage buckets](/boundary/docs/workers/filters#example-worker-filter-for-storage-buckets) to learn about worker tags and filters.

Reviewer: ```suggestion
   - **Worker filter**: (Required) A filter expression that indicates which Boundary workers have access to the storage. The filter must match an existing worker to create a Boundary storage bucket. It should be a boolean expression. To have Boundary format the filter for you, click the **Show filter generator** toggle. Refer to [Generate a filter in the Admin UI](/boundary/docs/workers/filters#generate-a-filter-in-the-admin-ui) for the procedure, or [the filter example for storage buckets](/boundary/docs/workers/filters#example-worker-filter-for-storage-buckets) to learn about worker tags and filters.
```
